lib/logstash/filters/translate.rb

```diff
@@ -81,7 +81,12 @@ class LogStash::Filters::Translate < LogStash::Filters::Base
 
   # When using a dictionary file, this setting will indicate how frequently
   # (in seconds) logstash will check the dictionary file for updates.
-  config :refresh_interval, :validate => :number, :default => 300
+  config :refresh_interval, :validate => :number, :default => 5
```

Comment:
Changing this will change the default behavior which means we'll need a major version bump.

I would prefer if this refresh_interval default was not changed.